Fix scrolling when using -webkit-overflow-scrolling:touch;

KZeni opened the pull request:
This is a great plugin, but I want my mobile visitors to still get the smooth scrolling that `-webkit-overflow-scrolling:touch;` provides. I applied that to the scrolling element's CSS, but it then made the headers not update properly and generally became glitchy.

I found that a quick fix to this was to have this set the headers to be `position:fixed;` rather than `position:absolute;`. The scrolling is now perfectly smooth and the headers still attach themselves as you'd expect.

I'd like to see this implemented in the main distribution of this since I'm sure other people will want to have sticky headers and smooth scrolling.

Thanks!

iclanzan commented:
I will review this as soon as I have some spare time. I remember there being no way of making it work with `-webkit-overflow-scrolling: touch` when I built this, but I think things have changed with iOS 8.

iclanzan commented:
I don’t think position fixed is a good work around though.

KZeni commented:
Based on the multiple things I've tried to get this to work, it seems to be the _only_ thing that addresses the issue and it's a small change. What makes it not a good workaround?

My thinking is, those elements are effectively position:fixed; when you consider their behavior, and setting it as such actually addresses a problem that's otherwise present when using smooth scrolling.

I'm not sure the hesitation to have the CSS set to be what they're effectively already doing and fix smooth scrolling. If it's a matter of browser compliance, then I'd consider adding this small change as an option that's set when you call the script (ex. `$('main').slinky({position:'fixed'});`) and let the developers out there determine if they're targeting devices that have issues with `position:fixed;`.
